securedrop/client.py

```diff
@@ -37,31 +41,86 @@ def make_dict(self):
         }
 
 
+class AESWrapper(object):
+
+    def __init__(self, key):
+        self.bs = AES.block_size
+        self.key = hashlib.sha256(key.encode()).digest()
+
+    def encrypt(self, raw):
+        verify = hashlib.sha256((raw.encode())).hexdigest()
```

Comment:
Why do we need a verification hash? Decryption will throw if it fails.

Reply:
ok.